sympy/vector/coordsysrect.py

```diff
@@ -98,6 +98,33 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
```

Comment:
What to do here:

- check that if `ŧransformation` is `None`.
- if it is None, computer `transformation` from the rotation and traslation. Make identity if they are also None.
- if it is not None, check that rotation and translation are None
- transform `transformation` to a `Lambda`.